packages/server/lib/server.ts

```diff
@@ -142,7 +142,7 @@ app.route('/admin/flow/deploy/pre-built').post(apiAuth, flowController.adminDepl
 app.route('/proxy/*').all(apiAuth, upload.any(), proxyController.routeCall.bind(proxyController));
 
 // Webapp routes (no auth).
-if (isCloud() || isEnterprise()) {
+if (AUTH_ADMIN_SWITCH_ENABLED) {
```

Comment:
This variable is incorrectly named for this context

Reply:
oops yes wrong c/c thanks